examples/redis/README.md

```diff
@@ -0,0 +1,228 @@
+# Redis
+
+[Redis](https://redis.io/) is the open source, in-memory data store used by millions of developers
+as a database, cache, streaming engine and message broker.
+
+## Introduction
+
+We will be using [Redis](https://github.com/bitnami/charts/tree/main/bitnami/redis) chart that bootstraps a Redis deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.20+
+- PV provisioner support in the underlying infrastructure
+- Kanister controller version 0.85.0 installed in your cluster, let's assume in Namespace `kanister`
+- Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
+
```

Comment:
As suggested above, building and pushing the the tools-image to a public repository could be a sub-heading under prereq maybe

Reply:
added steps to build and push docker image